src/lib.rs

```diff
@@ -19,6 +19,7 @@ pub mod cli_util;
 pub mod commands;
 pub mod config;
 pub mod diff_edit;
+mod diff_util;
```

Comment:
I haven't thought much about which functions to make public, but these functions actually seem useful for custom binaries to use, so maybe they should be public?